src/main/java/by/teachmeskills/repository/InDataUserRepository.java

```diff
@@ -0,0 +1,70 @@
+package by.teachmeskills.repository;
+
+import by.teachmeskills.data.DBUtil;
+import by.teachmeskills.module.User;
+import by.teachmeskills.module.UserEntityFields;
+
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+public class InDataUserRepository {
+
+    private InDataUserRepository() {
+    }
+
+    public static class InDataUserRepositoryHolder {
+        public static final InDataUserRepository HOLDER_INSTANCE = new InDataUserRepository();
+    }
+
+    public static InDataUserRepository getInstance() {
+        return InDataUserRepositoryHolder.HOLDER_INSTANCE;
+    }
+
+    private static final List<User> users = new ArrayList<>();
```

Comment:
Remove this field, create new collection in every method, this approach is not threadsafe